src/test/java/org/prebid/server/util/HttpUtilTest.java

```diff
@@ -133,4 +133,18 @@ public void cookiesAsMapShouldReturnExpectedResult() {
         assertThat(cookies).hasSize(1)
                 .containsOnly(entry("name", "value"));
     }
+
+    @Test
+    public void cookieToSetCookieShouldReturnCookieWithAdditionalParameter() {
```

Comment:
Please rename test to `cookieToHeaderValueShouldReturnExpectedString` and `setCookie` variable to `setCookieHeader`. Thanks!